scripts/create_tables.sql

```diff
@@ -1 +1,25 @@
 -- create doctors, patients, and appointments tables
+create table doctors (
+  id serial primary key,
+  family_name text,
+  given_name text,
+  gender text,
+  born_on date,
+  specialty text
+);
+
+create table patients (
+  id serial primary key,
+  family_name text,
+  given_name text,
+  gender text,
+  born_on date,
+  insurance text
+);
+
+create table appointments (
```

Comment:
This table is missing a `time` column and foreign key two columns, `doctor_id` and `patient_id`, referencing the `id`s of `doctors` and `patients`, respectively.